spec/unit/formatting.spec.js

```diff
@@ -51,6 +51,12 @@ describe("Formatting", function() {
             ).toBe("The quick brown <b>fox</b> jumps over the lazy <b>dog</b>.");
         });
     });
+    describe("stripIrcFormatting", function() {
+        it("should not strip ZWSP characters", () => {
```

Comment:
If there's a Zero Width Space in here, would you mind printing it as \u200B?
Otherwise I think it will get lost and a massive pain for the next person to figure out the `git diff`.

Reply:
Oh, right yes. That's an obvious one :facepalm: